.github/workflows/helm-release.yaml

```diff
@@ -147,3 +145,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ github.token }}
           RELEASE_VERSION: ${{ steps.get_version.outputs.RELEASE_VERSION }}
+
+      - name: Create Github Release + Tag
```

Comment:
this release now doesn't really do anything other than document the changes reflected in this release